policy/modules/services/resolvconf.fc

```diff
@@ -0,0 +1,5 @@
+/etc/resolvconf/update\.d/.*	--	gen_context(system_u:object_r:resolvconf_exec_t,s0)
+
+/usr/lib/resolvconf/.*	gen_context(system_u:object_r:resolvconf_exec_t,s0)
```

Comment:
Everything under here are resolvconf entrypoints? Seems too broad. If they're helpers then `bin_t` would be fine (moved to `corecommands.fc`)